### DISCUSSION TITLE
Which Non-Insider-Windows-Version will this be in?

### DISCUSSION THREAD

**Discussion by StefanWetterActiware** [microsoft/wslg#222] [General]
Hello everybody,

i am very excited to get this up and running. But i cannot use Windows Insider on my productive workstation...
Is a date for a release already known?

Regards,
Stefan

**Comment by boehs** [comment]
No obvious release date, but likely October 2021 in 21H2

Please see [here](https://github.com/microsoft/wslg/discussions/82#discussioncomment-675820)
